run_tests.sh

```diff
@@ -11,11 +10,10 @@ set shortmess+=I
 for dep in ['vader.vim', 'vim-textobj-user']
   execute 'set rtp+=' . finddir(dep, expand('~/.vim').'/**')
```

Comment:
I don't like this part of the original test setup. It looks for
dependencies in the user's normal vim environment.

That said, as far as I can tell, this file is not used by the CI.
Is that right? if so, I can easily edit this file later for local
test runs.

Does that make sense to you?

Reply:
Yes I don't like this either, and no it is not used by CI. I only messed with it to fix paths that I was changing not to actually overhaul it. It isn't even complete because it doesn't account for another dependency needed for tests (vim-surround). I would suggest something completely different as a local test runner anyway.